src/main.rs

```diff
@@ -405,7 +405,7 @@ fn run_path(path: &Path, special_crates: &[String]) {
 struct Opt {
     /// Special crates to put after third party crates, e.g. --special foo,bar,baz
     /// If none are given userp will attempt to use the workspace members of the root Cargo.toml
-    #[structopt(short, long)]
+    #[structopt(short, long, use_delimiter = true)]
```

Comment:
Wasn't `use_delimiter` deprecated? I have some sort of vague memory of this.

Reply:
Hmm. Not that I know of. It's still [documented in clap](https://docs.rs/clap/2.33.1/clap/struct.Arg.html#method.use_delimiter) (and usable as a [direct pass-through from structopt](https://docs.rs/structopt/0.3.15/structopt/#raw-methods)).